src/lucky_flow/expectations/have_text_expectation.cr

```diff
@@ -0,0 +1,19 @@
+struct LuckyFlow::Expectations::HaveTextExpectation
+  def initialize(@expected_value : String)
+  end
+
+  def match(element)
+    element.text == @expected_value
+  end
+
+  def failure_message(element)
+    <<-MSG
+    Expected element to have text: #{@expected_value}
+                            actual: #{element.text}
+    MSG
+  end
+
+  def negative_failure_message(element)
+    "Expected element not to have text: #{@expected_value}"
```

Comment:
What are your thoughts on saying *which* element in the error? Like

`Expected element <div> to not have text: Hi` or something? Would that just clutter things more?

Reply:
I was struggling with what the best error message would be. I'm not sure putting the tag name would be the best because it is not was they searched for. What about using the `raw_selector`? It wouldn't make much sense in that sentence though

Reply:
Ah, ok. Not a huge deal because we can always improve more later. This is a good start!